src/plone/restapi/tests/test_atcontent_serializer.py

```diff
@@ -193,3 +194,148 @@ def test_get_is_folderish_in_folder(self):
         obj = serializer()
         self.assertIn("is_folderish", obj)
         self.assertEqual(True, obj["is_folderish"])
+
+    def test_nextprev_disabled(self):
+        folder = api.content.create(
+            container=self.portal,
+            type="ATTestFolder",
```

Comment:
@rodfersou why are you using ATTestFolder here? How is that even possible with Python 3?

Reply:
@rodfersou Ok. Got it. This comes from our test layer. Though, it does not make any sense at all to use that here.

Reply:
this test file is about AT content types being serialized; thought would be good to see how the code behaves with AT content types.
still makes sense to me to keep as it is.

Reply:
Ok. I got that wrong then, sorry. The file test_atcontent_serializer.py is excluded from py3 tests.